app/views/devise/confirmations/new.html.erb

```diff
@@ -1,20 +1,47 @@
-<h2>Resend confirmation instructions</h2>
+<!-- ========== MAIN ========== -->
+<main id="content" role="main" class="border-bottom">
+  <!-- Resend confirmation form -->
+  <div class="container space-top-2 space-top-md-4 space-bottom-2">
+    <%= simple_form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post, class: "js-validate w-md-75 w-lg-50 mx-md-auto" }) do |f| %>
+      <!-- Title -->
+      <div class="mb-7">
+        <h1 class="h3 font-weight-normal mb-0">Didn't receive your email <span class="font-weight-semi-bold text-success">confirmation?</span></h1>
```

Comment:
I'd update to, "Didn't get your confirmation?" Probably without the green text on confirmation.